cmake/INSTALL.md

```diff
@@ -0,0 +1,34 @@
+# Install Instruction
+
+Execute following commands in the repo root.
+
+## Build with Old Style Make Generator
+```bash
+mkdir -p build && cd build
+cmake  -DCMAKE_INSTALL_PREFIX=../dist .. # configure
+cmake --build . --target install -- -j6  # build && install
+```
+
+## Build with Ninja Generator
+``` bash
+mkdir -p build && cd build
+cmake -GNinja -DCMAKE_INSTALL_PREFIX=../dist ..
+cmake --build . --target install
+```
+
+After built, you can find all installed files in <your_repo_root>/dist
+
+# For Advance User
```

Comment:
advanced users. Please clarify how you would supply those options.

Reply:
And I hope the current scripts still work?  E.g. there are no path changes required?

Reply:
Unfortunately, since we are doing an out of source build, it will not work out of box. But due to all executables are installed to a unified bin directory, it will be pretty easy to fix for the users. Pls see the updated file.